packages/tokens/carousel.ts

```diff
@@ -1,30 +1,29 @@
-import type { InjectionKey, Ref, ToRefs, UnwrapRef } from 'vue'
+import type { InjectionKey, Ref } from 'vue'
 
 import type { CarouselItemProps } from '@element-plus/components/carousel'
 
-export type CarouselItemContext = CarouselItemProps &
-  ToRefs<{
-    hover: boolean
-    translate: number
-    scale: number
-    active: boolean
-    ready: boolean
-    inStage: boolean
-    animating: boolean
-  }> & {
-    uid: number | undefined
-    translateItem: (
-      index: number,
-      activeIndex: number,
-      oldIndex: number
-    ) => void
-  }
+type CarouselItemStates = {
```

Comment:
Why not export?

Reply:
I see no reasons exporting this though, but still we can do that. I'll update it.